AnnoyancesFilter/sections/subscriptions.txt

```diff
@@ -41,6 +41,8 @@ fila.com#$#.signup-modal { display: none!important; }
 fila.com#$#.ui-widget-overlay { display: none!important; }
 fila.com#$#body { position: static!important; }
 gatdaily.com###custom_html-22
+jin115.com##aside + aside.section-box > .plugin-free_area
+jin115.com##.plugin-follow_buttons
```

Comment:
@zloyden Should follow button now be in subscription? (I don't see recommendation).

Reply:
The recommendation is `Follow`